security/providers/header/src/main/java/io/helidon/security/providers/header/HeaderAtnProvider.java

```diff
@@ -287,6 +297,9 @@ public Builder atnTokenHandler(TokenHandler tokenHandler) {
          * @param tokenHandler token handler instance
          * @return updated builder instance
          */
+        @ConfiguredOption(key = "outbound-token",
+                          description = "Token handler to create outbound headers to propagate identity. "
```

Comment:
description should be generated automatically from javadocs, no need to add it here